lib/MemoryFileSystem.js

```diff
@@ -16,10 +16,12 @@ class MemoryFileSystemError extends Error {
 	constructor(err, path) {
 		super(err, path);
 		if (Error.captureStackTrace)
-			Error.captureStackTrace(this, this.constructor)
+			Error.captureStackTrace(this, this.constructor);
 		this.code = err.code;
 		this.errno = err.errno;
 		this.message = err.description;
+		if (path)
+			this.message += ` "${path}"`;
```

Comment:
Could you align this with the node.js errors? They include the operation name in the message.